data/Hints/weekly.json

```
@@ -1,8 +1,9 @@
 {
     "name":                  "weekly",
     "gui_name":              "Weekly",
-    "description":           "Hint distribution for the weekly races. 6 Goal hints, 3 Barren hints, 6 Sometimes hints, 1 Dual hint, 4 Always hints.",
+    "description":           "Hint distribution for the weekly races: 5 Always, 6 Goal, 4 Barren at ToT, 5 Sometimes, 2 Dual, 30/40/50 hinted at Skull House.",
```

This is not part of the hint distro so it should be left out of the description.

```suggestion
    "description":           "Hint distribution for the weekly races: 5 Always, 6 Goal, 4 Barren at ToT, 5 Sometimes, 2 Dual.",
```

---

Never mind, Always hint counts are also not part of hint distro files and are included in descriptions regardless, so I guess this is fine as well.